interpreter/runtime/memory.mli

```diff
@@ -38,6 +38,8 @@ val load_packed :
 val load_simd_packed :
   pack_size -> pack_simd -> memory -> address -> offset -> value_type -> value
     (* raises Type, Bounds *)
+val load_simd_lane :
+  V128.t -> pack_size -> memory -> address -> offset -> value_type -> int (* lane index *) -> value
```

Comment:
It would be a bit more symmetric if the V128.t argument was a `value` (or vice versa, you replace `value` with `V128.t` here in the previous function).

Reply:
Can you replace the result `value` type, too?

Reply:
Sorry, I forgot to unresolve before, so you probably overlooked my previous reply here.

Reply:
Ah okay, done!